site/content/en/docs/commands/start.md

```diff
@@ -86,7 +86,7 @@ minikube start [flags]
       --uuid string                       Provide VM UUID to restore MAC address (hyperkit driver only)
       --vm                                Filter to use only VM Drivers
       --vm-driver driver                  DEPRECATED, use driver instead.
-      --wait strings                      comma separated list of kubernetes components to verify and wait for after starting a cluster. defaults to "apiserver,system_pods", available options: "apiserver,system_pods,default_sa,apps_running" . other acceptable values are 'all' or 'none', 'true' and 'false' (default [apiserver,system_pods])
+      --wait strings                      comma separated list of kubernetes components to verify and wait for after starting a cluster. defaults to "apiserver,system_pods,node_conditions", available options: "apiserver,system_pods,default_sa,apps_running,node_conditions,node_ready" . other acceptable values are 'all' or 'none', 'true' and 'false' (default [apiserver,system_pods,node_conditions])
```

Comment:
What's the difference between `node_conditions` and `node_ready`?

Can we pick names that are more meaningful?

Reply:
how about node_pressure and node_ready ?